Resources/Prototypes/Entities/Structures/Machines/holopad.yml

```
@@ -155,4 +155,674 @@
     scrollRate: 0.125
   - type: Tag
     tags:
-    - HideContextMenu
\ No newline at end of file
+    - HideContextMenu
+
+## Mapping prototypes
+# General
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralTools
+  suffix: Tools
+  components:
+  - type: Label
+    currentLabel: General - Tools
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralCryosleep
+  suffix: Cryosleep
+  components:
+  - type: Label
+    currentLabel: General - Cryosleep
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralTheater
+  suffix: Theater
+  components:
+  - type: Label
+    currentLabel: General - Theater
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralDisposals
+  suffix: Disposals
+  components:
+  - type: Label
+    currentLabel: General - Disposals
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralEVAStorage
+  suffix: EVA Storage
+  components:
+  - type: Label
+    currentLabel: General - EVA Storage
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralLounge
+  suffix: Lounge
+  components:
+  - type: Label
+    currentLabel: General - Lounge
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralArcade
+  suffix: Arcade
+  components:
+  - type: Label
+    currentLabel: General - Arcade
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralEvac
+  suffix: Evac
+  components:
+  - type: Label
+    currentLabel: General - Evac
+
+- type: entity
+  parent: Holopad
+  id: HolopadGeneralArrivals
+  suffix: Arrivals
+  components:
+  - type: Label
+    currentLabel: General - Arrivals
+
+# Command
+- type: entity
+  parent: Holopad
+  id: HolopadCommandBridge
+  suffix: Bridge
+  components:
+  - type: Label
+    currentLabel: Command - Bridge
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandVault
+  suffix: Vault
+  components:
+  - type: Label
+    currentLabel: Command - Vault
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandBridgeHallway
+  suffix: Bridge Hallway
+  components:
+  - type: Label
+    currentLabel: Command - Bridge Hallway
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandMeetingRoom
+  suffix: Command Meeting
+  components:
+  - type: Label
+    currentLabel: Command - Meeting Room
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandLounge
+  suffix: Command Lounge
+  components:
+  - type: Label
+    currentLabel: Command - Lounge
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandCaptain
+  suffix: Captain
+  components:
+  - type: Label
+    currentLabel: Command - Captain
+
+- type: entity
+  parent: Holopad
+  id: HolopadCommandHOP
```

For names like these, I'm not sure if it conflicts with our naming conventions or not. I know stuff like the AME has to be named `ControllerAme` for example.

---

you mean HolopadCommandHeadOfPersonnel in this case?

---

well, if it followed naming conventions, it would be
```suggestion
  id: HolopadCommandHop
```
no?

---

I'll fix that when i fix locale as well then :godo: